index.js

```diff
@@ -35,29 +33,18 @@ function licensee (configuration, path, callback) {
   ) {
     callback(new Error('No licenses or packages allowed.'))
   } else {
-    // Delete node_modules/.package-lock.json, Arborist/npm's
-    // on-disk cache of the resolved package tree, if
-    // present. When this is present, it may or may
-    // not include license metadata for packages. See
-    // https://github.com/jslicense/licensee.js/issues/64#issuecomment-1145256328=
-    fs.rm(
-      joinPath(path, 'node_modules', '.package-lock.json'),
-      { force: true },
-      function (/* ignore errors */) {
-        var arborist = new Arborist({ path })
-        arborist.loadActual()
-          .catch(function (error) {
-            return callback(error)
-          })
-          .then(function (tree) {
-            var children = Array.from(tree.children.values())
-            if (configuration.filterPackages) {
-              children = configuration.filterPackages(children)
-            }
-            callback(null, findIssues(configuration, children, []))
-          })
-      }
-    )
+    var arborist = new Arborist({ path })
+    arborist.loadActual({ forceActual: true })
+      .catch(function (error) {
+        return callback(error)
+      })
```

Comment:
if this doesn't rethrow, it will trigger the next `.then` - so you may need to move it until after the then

Reply:
this doesn't seem resolved yet